rca/people/admin/__init__.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib import admin
+from import_export import widgets
+from import_export.fields import Field
+from wagtail.core.models import Page
+
+from ..models import StaffPage
+from .page_import import PageImportMixin, PageResource
+
+
+class StaffPageResource(PageResource):
+    legacy_staff_id = Field(
+        attribute="legacy_staff_id",
+        column_name="id",
+        default=None,
+        widget=widgets.IntegerWidget(),
+    )
+    staff_title = Field(
+        attribute="staff_title",
+        column_name="title_prefix",
+        default="",
+        widget=widgets.CharWidget(),
+    )
+
+    class Meta:
+        model = StaffPage
+        # If not set, diffing pages triggers 'pickle' errors
+        skip_diff = True
+        # Pages clean themselves automatically on save()
+        clean_model_instances = False
+        import_id_fields = ("legacy_staff_id",)
+        fields = (
+            "legacy_staff_id",
+            "title",
+            "slug",
+            "staff_title",
+            "first_name",
+            "last_name",
+        )
+
+    def before_save_instance(self, instance, using_transactions=True, dry_run=False):
+        """
+        Add missing values for 'title' and 'job_title', as they are required.
```

Comment:
Guess we can rove job title from this

Reply:
Ah yes :) Always forgetting the docstrings